index.html

```diff
@@ -104,8 +104,7 @@ <h2><code>jiant</code></h2>
 <li><code>jiant</code> contains implementations of strong baselines for the <a href="https://gluebenchmark.com">GLUE</a> and <a href="https://gluebenchmark.com/super">SuperGLUE</a> benchmarks, and it's the recommended starting point for work on these benchmarks.</li>
```

Comment:
Somewhere on this page, we should make it clear which version is recommended as a GLUE/SuperGLUE competition entry starting point? We should switch that to v2, but that requires testing that we can make legal submissions with okay scores on both benchmark sites.